packages/fast-components-styles-msft/src/utilities/acrylic.ts

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
lets re-enable max-line-length after this

Reply:
good catch!